### DISCUSSION TITLE
[github mode] Is it possible to change the default branch from `main` to another one ?

### DISCUSSION THREAD

**Discussion by notjb** [Thinkmill/keystatic#829] [Q&A] [answered]
Love keystatic, it works really well 💪

Tho the default branch being `main` bothers me.
Am i missing something in the config steps regarding changing the default branch from main to another one ?

Thanks a bunch !

**Comment by emmatown** [comment, marked as answer]
Keystatic uses whatever default branch is configured in GitHub, if you'd like to change it, you can change it through GitHub

**Reply by notjb** [reply]
Thanks a lot ! It wasn't clear to me the repo config would impact on this !
